### ISSUE TITLE
Actually compute pixel distance to camera center in Long-Term DL1 check

### ISSUE THREAD

**Issue by maxnoe** [cta-observatory/cta-lstchain#1074]
One plot shows cosmic pulses vs. pixel-id, implying that one should see a falling trend with the pixel id.

This somewhat works as the pixel id is sorted by the spiral pattern but there is also a "sub-pattern" due to the hexagonal spirals.

We should just compute the pixel's distance to the camera center as `r = np.sqrt(cam.pix_x**2 + cam.pix_y**2)` and use that for plotting

**Comment by moralejo** [comment]
The purpose of the plots is just to facilitate finding the value of a quantity for a given pixel id, if needed. 
Given the spiral numbering they also show roughly the dependency of the quantity vs. distance to camera center, but that is just a side effect. 
Using the true distance would still be a bit misleading for outermost pixels, for a given distance-to-center some pixels are at the edge of the camera while others are still well inside (towards the corners).

**Comment by maxnoe** [comment]
There is a border that shows the expected interval, which heavily implies there is a connection to the distance...

**Comment by moralejo** [comment]
Yes, those are just rough limits which are relaxed near the border because of edge effects.

And indeed, the limits in those plots are only valid for LST's spiral pixel numbering which more or less correlates with distance.  Certainly for a generic pixel numbering this would have to be modified.
